Cargo.toml

```diff
@@ -100,5 +100,7 @@ rustc-std-workspace-std = { path = 'library/rustc-std-workspace-std' }
 # source code for this crate.
 backtrace = { path = "library/backtrace" }
 
+hashbrown = { git = "https://github.com/rust-lang/hashbrown", rev = "8bee6b8" }
```

Comment:
You can point this to your branch for the perf run.

Reply:
The last run was explicitly to test hashbrowns current master, otherwise I pointed it to my branch. Since the last run looked good it could be a good idea to publish a patch release of hashbrown so the first optimization PR can be included in rustc.

Reply:
I think you should continue using this PR to test the perf impact of your changes in hashbrown and hopefully resolve the remaining regressions.

Reply:
Ok, but to be clear the first, merged PR (https://github.com/rust-lang/hashbrown/pull/204) does not show any regressions. It is only the second (https://github.com/rust-lang/hashbrown/pull/205) that regressed.